build.sbt

```diff
@@ -65,6 +70,15 @@ val coreDependencies = Seq(
   "javax.validation" % "validation-api" % "1.1.0.Final" force()
 )
 
+val loggingDependencies = Seq(
+  "ch.qos.logback" % "logback-core" % "1.1.2",
+  "ch.qos.logback" % "logback-classic" % "1.1.2",
```

Comment:
is logback only necessary for the server? If so we might not want to put it into core tranquility